backend/codesirius/problems/serializers/language.py

```
@@ -43,6 +39,14 @@ def create(self, validated_data):
             language = Language.objects.create(**validated_data)
             logger.info("Language created successfully")
             return language
+        except IntegrityError:
+            logger.error("Language with the same name and version already exists")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)